lib/api.definition.js

```diff
@@ -17,23 +16,8 @@ api.definition.require = (
   return definition;
 };
 
-api.definition.walk = (
-  // Walk through both data tree and definition tree
-  // and call function for each data item
-  data, // data to be validated
-  definition, // validation rules
-  name, // string
-  callback // function(parent, data, definition, name, path)
-  //   parent - item parent
-  //   data - certain item
-  //   definition - data structures definition hash
-  //   name - item name in parent
-  //   path - path to current item
-  //   def - parsed data definition
-  // Return: { valid, errors }
-  //   valid - boolean is result valud
-  //   errors - array or empty array
-) => {
+// Walk through both data tree and definition tree
+api.definition.walk = (data, definition, name, callback) => {
```

Comment:
Was this comment removal intentional?

Reply:
We will rewrite comments later, all old comments requires review